src/core/server/http/http_config.ts

```diff
@@ -21,8 +23,7 @@ import {
 } from './security_response_headers_config';
 
 const validBasePathRegex = /^\/.*[^\/]$/;
```

Comment:
I can see us moving this regex to `core-base-server-internal` too.

Reply:
This one is not exported, and only used internally within this file, so it felt unnecessary to move it. But I can if you want.

Reply:
Thanks for moving it! We can always move it back during the cleanup phase if we decide to isolate it.